ui/src/app/views/notebook-redirect/component.ts

```diff
@@ -125,16 +122,15 @@ export class NotebookRedirectComponent implements OnInit, OnDestroy {
   ) {}
 
   ngOnInit(): void {
-    this.wsNamespace = this.route.snapshot.params['ns'];
-    this.wsId = this.route.snapshot.params['wsid'];
-    this.creating = this.route.snapshot.queryParams['creating'] || false;
-    this.playground = (this.route.snapshot.queryParams['playgroundMode'] === 'true');
-    this.jupyterLabMode = (this.route.snapshot.queryParams['jupyterLabMode'] === 'true');
+    const {ns, wsid} = urlParamsStore.getValue();
+    const {creating, playgroundMode, jupyterLabMode, kernelType} = queryParamsStore.getValue();
```

Comment:
Do you need to do the Kernels conversion as we saw below? `Kernels[kernelType]`

Reply:
`this.kernelType` isn't actually used, so I got rid of it.